albumentations/augmentations/bbox_utils.py

```diff
@@ -119,29 +119,39 @@ def convert_bbox_to_albumentations(bbox, source_format, rows, cols, check_validi
 
     Args:
         bbox (list): bounding box
-        source_format (str): format of the bounding box. Should be 'coco' or 'pascal_voc'.
+        source_format (str): format of the bounding box. Should be 'coco', 'pascal_voc', or 'yolo'.
         check_validity (bool): check if all boxes are valid boxes
         rows (int): image height
         cols (int): image width
 
     Note:
         The `coco` format of a bounding box looks like `[x_min, y_min, width, height]`, e.g. [97, 12, 150, 200].
         The `pascal_voc` format of a bounding box looks like `[x_min, y_min, x_max, y_max]`, e.g. [97, 12, 247, 212].
+        The `yolo` format of a bounding box looks like `[x, y, height, width]`, e.g. [0.3, 0.1, 0.05, 0.07].
 
     Raises:
-        ValueError: if `target_format` is not equal to `coco` or `pascal_voc`.
+        ValueError: if `target_format` is not equal to `coco` or `pascal_voc`, ot `yolo`.
 
     """
-    if source_format not in {'coco', 'pascal_voc'}:
+    if source_format not in {'coco', 'pascal_voc', 'yolo'}:
         raise ValueError(
-            "Unknown source_format {}. Supported formats are: 'coco' and 'pascal_voc'".format(source_format)
+            "Unknown source_format {}. Supported formats are: 'coco', 'pascal_voc' and 'yolo'".format(source_format)
         )
     if source_format == 'coco':
         x_min, y_min, width, height = bbox[:4]
         x_max = x_min + width
         y_max = y_min + height
+    elif source_format == 'yolo':
+        # https://github.com/pjreddie/darknet/blob/f6d861736038da22c9eb0739dca84003c5a5e275/scripts/voc_label.py#L12
+        x, y, width, height = denormalize_bbox(bbox[:4], rows, cols)
```

Comment:
Since YOLO format assumest normalized coordinates, for sake of sanity checks, can you please add assertions that values of `bbox[:4]` are floating-points here, not integers? This would help users quickly realize they are using YOLO keypoints in a wrong way.

Reply:
done